game.cpp

```diff
@@ -858,7 +903,7 @@ void test_game() //!OCLINT tests may be many
 
     assert(has_collision(g));
   }
-  //#define FIX_ISSUE_233
+#define FIX_ISSUE_233
```

Comment:
These precompiler statements (i.e. those that start with `#`) can be removed when the test passes. I'll let a newbie do it :-)